bot/index.js

```diff
@@ -65,7 +65,7 @@ async function init() {
         }
     }
 
-    global.discordClient._connectToDiscord = connectToDiscord;
+    global.discotron._connectToDiscord = connectToDiscord;
```

Comment:
Should it be renamed to `connectToDiscord` without the `_`, now that it is a global object that we have full control over its properties?

Reply:
I think we should leave it, because this should disappear anyway in the future.